schemas/context/webpage.schema.json

```diff
@@ -0,0 +1,103 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/webpage",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Web Page",
+  "type": "object",
+  "description":
+    "Details about the web page where a web interaction occurred, as recorded by an `ExperienceEvent`.",
+  "definitions": {
+    "webpage": {
+      "properties": {
+        "xdm:siteSection": {
+          "title": "Site Section",
+          "type": "string",
+          "description":
+            "The normative name of the site section where this web page resides, which may be used to classify or categorize the interaction"
+        },
+        "xdm:name": {
+          "title": "Name",
+          "type": "string",
+          "description":
+            "The normative name of the web page. This name is not necessarily the page title or directy associate with page content, but is used to organize a site's pages for classification purposes"
+        },
+        "xdm:URL": {
+          "title": "URL",
+          "type": "string",
+          "format": "uri",
+          "description":
+            "The normative or usual URL of the web page.  This may or may not be the actual URL used to reach the page, which would be recorded using `WebLink`."
+        },
+        "xdm:server": {
+          "title": "Server",
+          "type": "string",
+          "format": "hostname",
+          "description":
+            "The normative or usual server that hosts the web page.  This may or may not be the host or server that actually served the page interaction, but is used for classification purposes."
```

Comment:
Again, I'd put the focus not on which server is doing the hosting, but under which hostname the experience can be accessed.

The fact that there is a real (technical) and virtual (used for classification purposes) hostname is fine.